spring-integration-mqtt/src/main/java/org/springframework/integration/mqtt/core/ClientManager.java

```diff
@@ -50,7 +52,17 @@ public interface ClientManager<T, C> extends SmartLifecycle, MqttComponent<C> {
 	 * Return the managed client.
 	 * @return the managed client.
 	 */
-	T getClient();
+	@Nullable T getClient();
+
+	/**
+	 * @return the managed client url, never null
+	 */
+	String getUrl();
+
+	/**
+	 * @return the managed clientId, never null
+	 */
+	String getClientId();
```

Comment:
Please, add, to these new public API `@since 7.0` and some short general sentence.
Could be just copy of that `@return` tag content.